code/AuditHook.php

```diff
@@ -393,10 +395,13 @@ public function authenticationFailed($data)
     }
 
     /**
-     * @deprecated 2.1...3.0 Use tractorcow/silverstripe-proxy-db instead
+     * @deprecated 2.1.0 Use tractorcow/silverstripe-proxy-db instead
      */
     public function onBeforeInit()
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('2.1.0', 'Use tractorcow/silverstripe-proxy-db instead');
+        });
```

Comment:
This method call means we need to bump the framework constraint

Reply:
Updated